src/onnxruntime.cc

```diff
@@ -264,6 +266,9 @@ ModelState::LoadModel(
                           param_key.c_str(), &value_string));
                       if (value_string == "FP16") {
                         trt_options.trt_fp16_enable = 1;
+                      } else if (value_string == "INT8") {
+                        trt_options.trt_int8_enable = 1;
+                        trt_options.trt_int8_calibration_table_name = &int8_calibration_table_name;
```

Comment:
```suggestion
                        trt_options.trt_int8_calibration_table_name = int8_calibration_table_name.c_str();
```
trt_int8_calibration_table_name seems to be const char* based on the definition of OrtTensorRTProviderOptions

Reply:
I'm surprised that the compile didn't fail due to the type mismatch

Reply:
I did not pick the changes correctly ... I had staged this file already and then added this fix after seeing compilation error... but forgot to stage this fix before the commit...